pyro/distributions/torch_patch.py

```diff
@@ -53,7 +53,9 @@ def _Transform__init__(self, cache_size=0):
 def _Transform__getstate__(self):
     attrs = {}
     for k, v in self.__dict__.items():
-        if not isinstance(v, weakref.ref):
+        if isinstance(v, weakref.ref):
```

Comment:
I realized that it is important to serialize this when I ran the baseball example on CUDA using multiprocessing, so that `transform._inv` is None and recomputed but doesn't raise an AttributeError.